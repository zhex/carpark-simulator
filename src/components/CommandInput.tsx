import * as React from 'react';
import styled from 'styled-components';

const CommandInput = styled.textarea`
    display: block;
    height: 200px;
    width: 100%;
    font-size: 14px;
    margin-bottom: 20px;
`;

export default CommandInput;
