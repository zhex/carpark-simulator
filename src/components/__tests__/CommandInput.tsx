import 'jest-styled-components';
import * as React from 'react';
import { create } from 'react-test-renderer';

import CommandInput from '../CommandInput';

it('CommandList Component', () => {
    const tree = create(<CommandInput />).toJSON();
    expect(tree).toMatchSnapshot();
});
