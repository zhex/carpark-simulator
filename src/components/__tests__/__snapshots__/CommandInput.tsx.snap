// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CommandList Component 1`] = `
.c0 {
  display: block;
  height: 200px;
  width: 100%;
  font-size: 14px;
  margin-bottom: 20px;
  resize: none;
}

<textarea
  className="c0"
/>
`;
